prometheus_client/core.py

```diff
@@ -844,6 +851,11 @@ def _floatToGoString(d):
         return '-Inf'
     elif math.isnan(d):
         return 'NaN'
+    elif sys.version_info[:2] == (2, 6):
```

Comment:
Why do you need this?

Reply:
It was todo with trying to fix that fact that repr of float in 2.6 is different. Ended up just skipping those tests on 2.6.